packages/adapters/gatewallet/src/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { isInMobileBrowser } from '@tronweb3/tronwallet-abstract-adapter';
+
+export function supportGateWallet() {
+    return !!(window.gatewallet && window.gatewallet.tronLink);
```

Comment:
Is this function valid? I test the demo in GateWallet Ios app and browser with gate extension, but there is no `window.gatewallet.tronLink` .